BikeBattle/app/src/main/java/edu/hm/cs/bikebattle/app/activities/MainActivity.java

```diff
@@ -121,6 +132,29 @@ public void error(int error, Throwable exception) {
   }
 
   @Override
+  public void refreshUserInfo() {
+
+    final User user = getPrincipal();
+    final String name = user.getName();
+    final String email = user.getEmail();
+    final Uri foto = getUserPhoto();
+
+    runOnUiThread(new Runnable(){
```

Comment:
Nein, du machst einfach in der Base eine leere Methode und wenn eine abgeleitete Activity was mit dem Principle machen will, überschreibt die Activity diese Methode.